### DISCUSSION TITLE
Having an AI leader

### DISCUSSION THREAD

**Discussion by shirinrs** [WeAreMahsaAmini/Core#460] [Ideas]
Could we make an AI leader? Is it possible to create a virtual leader who can anticipate everything?

**Comment by arashi-dev** [comment]
Developing an AI needs a team of expert developers with a lot of experience. and it takes years and a lot of money to develop an AI.

But maybe #461 is possible 🤔

**Comment by bidar-ca** [comment]
Interesting

I like your idea

**Comment by babooloo** [comment]
What would be the training data?
